addons/account_edi/models/account_edi.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from odoo.tools.pdf import OdooPdfFileReader, OdooPdfFileWriter
+
+from lxml import etree
+import base64
+import io
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdiFormat(models.Model):
+    _name = 'account.edi.format'
+    _description = 'EDI format'
+
+    name = fields.Char(index=True)
+    code = fields.Char()
+    export_invoice = fields.Selection(selection=[
+        ('embed_to_pdf', 'Embed in PDF'),
+        ('separated_attachment', 'Attachment'),
+        ('not_supported', 'Not Supported')  # only import is supported
+    ], default='embed_to_pdf')
```

Comment:
@qdp-odoo We could remove this field and add a method to be overridden instead. What do you think? It could add more flexibility